assets/css/_route_picker.scss

```diff
@@ -0,0 +1,52 @@
+.m-route-picker {
+  background-color: $light-periwinkle;
+  height: 100%;
+  overflow-y: scroll;
+  padding: 1rem;
+  position: fixed;
+  width: 7.5rem;
+  z-index: 2;
+}
+
+.m-route-picker__selected-routes,
+.m-route-picker__route-list {
+  list-style-type: none;
+  padding: 0;
+}
+
+.m-route-picker__selected-routes {
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: 2rem;
+  margin-top: 0;
+  min-height: 2rem;
+}
+
+.m-route-picker__selected-routes-button,
+.m-route-picker__route-list-button {
+  border: none;
+  border-radius: 4px;
+  color: $dark;
+  font-family: $font-family;
+  font-size: 0.8125rem;
+  font-weight: bold;
+  padding: 0.3125rem;
+}
```

Comment:
Do we anticipate wanting this to be a reusable style for all buttons?

Reply:
I might suggest waiting for a design library before making calls like that. Right now 2 of 4 buttons share only some of their styles. I think that pulling these out could be reasonable, but I'd suggest doing that in combination with pulling out a Button component. Personally I think it works well to have a one-to-one mapping between a component file and a css file.